docs/vcf_files_preprocessor.md

```diff
@@ -20,9 +20,15 @@ This tool generates a report that includes three types of inconsistencies:
 * Malformed Records (Optional): There can be malformed records that cannot be
   parsed correctly by the VCF parser. In this case, the file paths and the
   malformed variant records are included in the report for reference.
-* Undefined headers (Optional): Occasionally, some fields may be used directly
-  without a definition in the VCF files. The tool can infer the type for such
-  fields and provides them in the report.
+* Inferred headers (Optional): Occasionally, some fields may be used directly
+  without a definition in the VCF files, or though a definition is provided, the
+  field value does not match the field definition. This tool can infer the type
+  and num for such fields and provides them in the report. For the latter case,
+  two kinds of the mismatches are handled:
+  1. The defined field type is `Integer`, but the provided value is float.
+     Correct the type to be `Float`.
+  2. Defined num is `A`, but the provided values do not have the same
+     cardinality as the alternate bases. Infer the num to be `None`.
```

Comment:
nit: s/None/unknown (i.e. `.`)
(None is a PyVCF concept).

Reply:
Thank you for the info!